homeassistant/components/traccar/device_tracker.py

```diff
@@ -93,7 +93,8 @@
         config.get(CONF_SCAN_INTERVAL, SCAN_INTERVAL),
         config[CONF_MONITORED_CONDITIONS], config[CONF_EVENT])
 
-    return await scanner.async_init()
+    await scanner.async_init()
+    return True
```

Comment:
We should check credentials and return False if they are bad.

Reply:
We can only test that if the server is available, added a check for it now.